services/web-app/stories/components/type-weight.stories.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { TypeWeight } from '@/components/type'
+
+const stories = {
+  title: 'Platform/Components/TypeWeight',
+  component: TypeWeight,
+  argTypes: {},
+  parameters: {}
+}
+
+export default stories
+
+const Template = (args) => <TypeWeight {...args} />
+
+export const Default = Template.bind({})
+Default.args = {}
+
+export const Italic = Template.bind({})
+Italic.args = {
+  type: 'italic'
+}
+
+export const Types = Template.bind({})
+Types.args = {
+  type: 'types'
+}
```

Comment:
Do we need to have these three stories if there's a control to switch the type?
<img width="1128" alt="image" src="https://user-images.githubusercontent.com/40550942/166073889-464aadf2-26be-46cd-b7b2-6d8f14019f31.png">